### PR TITLE
adjusts text align on contact page

### LINKED ISSUE #251: Adjust the text-align for the contact page on mobile

### PR DESCRIPTION
**Pull request by sssoja**
### What changes have you made?
- changed the text-align on the contact page to justify across all screen sizes in line with the look and feel of the rest of the site  
- refactored the code a bit removing any unnecessary props 

### Which issue(s) does this PR fix?

Fixes #251 

### Screenshots (if there are design changes)

Screenshot 2020-11-24 at 16 13 56

### How to test
go to http://localhost:3000/contact and make sure display looks good on desktop and mobile

### CODE REVIEW

File: src/components/Contact.tsx [hunk 1]
```diff
@@ -37,30 +31,23 @@ const Contact = () => {
...
-            >
-              {t("contact.email")}
-            </a>
-          </ContactInfo>
-          <ContactInfo fontSize={fontSizes}>
-            {t("contact.location")}
-          </ContactInfo>
-        </ul>
+      <Container textAlign="justify">
+        <H2 fontSize={[4, 5, 6, 7]} py={3}>
```
Comment (mantagen) [inline src/components/Contact.tsx:35]: Surely this is the H1? Title of the page

Comment (sssoja) [reply src/components/Contact.tsx]: yah, honestly I don't know what came over me, I started questioning everything about semantic html. think it was time to step away from the code at this point

**Review by Jbarget — APPROVED** [review]

File: src/components/Contact.tsx [hunk 2]
```diff
@@ -37,30 +31,23 @@ const Contact = () => {
...
+      <Container textAlign="justify">
+        <H1 fontSize={[4, 5, 6, 7]} py={3}>
+          {t("contact.header")}
+        </H1>
+        <H2 fontSize={fontSizes} py={3}>
+          {t("contact.subheader")}
+        </H2>
+        <ContactInfo fontSize={fontSizes} py={1}>
+          <a
+            href="mailto:ox@oxymore.com"
```
Comment (Jbarget) [inline src/components/Contact.tsx:43]: this email should be `@oxymoremagazine.com`?

Comment (sssoja) [reply src/components/Contact.tsx:43]: I've emailed Aitor

**Pull request merged**